src/app/containers/app/app.component.ts

```diff
@@ -78,7 +82,7 @@ export class AppComponent implements OnInit, OnDestroy {
   }
 
   public handleCookieBannerFeatureToggle(): void {
-    this.cookieBannerEnabledSubscription = this.featureToggleService.isEnabled('mc-cookie-banner-enabled')
+    this.cookieBannerEnabledSubscription = this.featureService.isEnabled('mc-cookie-banner-enabled')
```

Comment:
Is this not already in master? Maybe you need to rebase/merge?

Reply:
Its just a rename of featureToggleService to featureService

Reply:
Ah yes, of course. My bad.